recipes/googletrans/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "googletrans" %}
+{% set version = "2.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "49cce281c5ded86fc1ef0429e240da650bae2dd7939dc5e664d80c33fa8ba3b0" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://files.pythonhosted.org/packages/49/0f/eac53560e99a6b1e3b3e18e8b98f0e0041bfc4f1ee1e1b74cdf5530786d0/googletrans-2.3.0.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
+  preserve_egg_dir: True
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
```

Comment:
I think you may only need `python`, `setuptools` and `pip` in the `host` section.